src/common/api.ts

```diff
@@ -100,6 +101,7 @@ export const initializeApi = (
 
       config.headers.common['X-WZ-API-KEY'] = apiKey;
       config.headers.common['X-WZ-API-SECRET'] = apiSecret;
+      config.headers.common['Referer'] = Referer;
```

Comment:
Peux-tu ajouter un commentaire pour expliquer l'utilité de ce paramètre ?

```suggestion
     // Only effective on server environment. This settings will be overiden by client browser.
      config.headers.common['Referer'] = Referer;
```